### ISSUE TITLE
wrong focus shift

### ISSUE THREAD

**Issue by charlesay** [zpydr/gnome-shell-extension-taskbar#82]
When there is more than one window open on a workspace, clicking on the icon for one of them in some cases fails to bring it into keyboard focus, instead putting focus on the most recently opened window.

**Comment by zpydr** [comment]
Fixed.
